examples/look/look/app.py

```diff
@@ -2,7 +2,8 @@
 
 import falcon
 
-from .images import ImageStore, Resource
```

Comment:
Same as for ASGI, we need to recheck if we don't need to sync the copypasted copies.